paasta_tools/cli/cmds/status.py

```diff
@@ -147,19 +149,17 @@ def report_status_for_cluster(service, cluster, deploy_pipeline, actual_deployme
 
         # Case: service deployed to cluster.instance
         if namespace in actual_deployments:
-            formatted_instance = PaastaColors.blue(instance)
-            version = actual_deployments[namespace][:8]
-            # TODO: Perform sanity checks once per cluster instead of for each namespace
-            status = execute_paasta_serviceinit_on_remote_master('status', cluster, service, instance,
-                                                                 system_paasta_config, verbose=verbose)
+            deployed_instances.append(instance)
+
         # Case: service NOT deployed to cluster.instance
         else:
-            formatted_instance = PaastaColors.red(instance)
-            version = 'None'
-            status = None
+            print '  instance: %s' % PaastaColors.red(instance)
```

Comment:
It is hard to read from the diff, but does this mean it prints all the instance names and git shas, and *then* prints that status for all of them?

I would rather it be interweaved. (where the server component does the printing)